libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -211,6 +218,13 @@ io::sstream& CodeGenerator::generateShaderInputs(io::sstream& out, ShaderType ty
             generateDefine(out, "LOCATION_BONE_WEIGHTS", uint32_t(VertexAttribute::BONE_WEIGHTS));
         }
 
+        for (int i = 0; i < MAX_CUSTOM_ATTRIBUTES; i++) {
+            if (!attributes.test(VertexAttribute::CUSTOM0 + i)) {
+                continue;
```

Comment:
Why not `if (attributes.test…) { generateIndexedDefine(…) }` instead of using a continue?

Reply:
Fixed